FlowCrypt/Functionality/Services/Local Pub Key Services/LocalContactsProvider.swift

```diff
@@ -42,8 +43,17 @@ extension LocalContactsProvider: LocalContactsProviderType {
     }
 
     func retrievePubKeys(for email: String) -> [String] {
-        find(with: email)?.pubKeys
-            .map { $0.armored } ?? []
+        find(with: email)?.pubKeys.map(\.armored) ?? []
+    }
+
+    func findBy(longid: String) async -> RecipientWithSortedPubKeys? {
```

Comment:
I don't think this is used anywhere in this PR - unless I overlooked

Reply:
Yes, it was used in initial code for signature verification, but not needed anymore, removed it